test/JDBC/upgrade/latest/schedule

```diff
@@ -114,6 +114,7 @@ BABEL-2944
 BABEL-3116
 BABEL-3118
 BABEL-3249
+BABEL-3486
```

Comment:
Can you specify in the commit message that TRY_CONVERT was incorrectly marked as an unsupported feature? Also please update your PR message with the new commit message.